go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/cloudability/metrics-agent
+
+go 1.13
+
+require (
+	github.com/golangci/golangci-lint v1.18.0
+	github.com/sirupsen/logrus v1.4.3-0.20190701143506-07a84ee7412e
```

Comment:
v1.4.3-0.20190701143506-07a84ee7412e introduces PadLevelText which we are using .. it has been merged to master but they have not cut a release since May 2019.